src/Teams/Data/Migrations/20200910113553_DataLayerAdded.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Teams.Data.Migrations
+{
+    public partial class DataLayerAdded : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "team",
+                columns: table => new
+                {
+                    id = table.Column<int>(nullable: false)
+                        .Annotation("SqlServer:Identity", "1, 1"),
+                    team_name = table.Column<string>(nullable: true),
```

Comment:
In real all this columns required
Prefer to have not null for all columns

Reply:
Fixed that.